WordPress/Classes/ViewRelated/NUX/LoginFields.h

```diff
@@ -41,5 +41,7 @@ NS_ASSUME_NONNULL_BEGIN
                            userIsDotCom:(BOOL)userIsDotCom
                shouldDisplayMultiFactor:(BOOL)shouldDisplayMultifactor;
 
+- (NSDictionary *)helpshiftLoginOptions;
```

Comment:
This is a good idea!

I wonder if `sourceTag` should be moved here? 🤔 Something to think about for the future, anyway!